src/app/modules/cadastre/cadastre-ori-search-tool/cadastre-ori-search-tool.component.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
The class name refers to the client namespace instead of the cadastre namespace. Also, you probably don't need the outer div for now.